packages/jest-matchers/src/__tests__/__snapshots__/matchers-test.js.snap

```diff
@@ -1812,13 +1812,13 @@ Received:
   [31m[Function anonymous][39m"
 `;
 
-exports[`.toEqual() expect({"a": 1, "b": [Function b], "c": true}).not.toEqual({"a": 1, "b": {"expectedObject": [Function Function]}, "c": {}}) 1`] = `
+exports[`.toEqual() expect({"a": 1, "b": [Function anonymous], "c": true}).not.toEqual({"a": 1, "b": {"expectedObject": [Function Function]}, "c": {}}) 1`] = `
 "[2mexpect([22m[31mreceived[39m[2m).not.toEqual([22m[32mexpected[39m[2m)[22m
 
 Expected value to not equal:
   [32m{\"a\": 1, \"b\": {\"expectedObject\": [Function Function]}, \"c\": {}}[39m
 Received:
-  [31m{\"a\": 1, \"b\": [Function b], \"c\": true}[39m"
+  [31m{\"a\": 1, \"b\": [Function anonymous], \"c\": true}[39m"
```

Comment:
no it is not. Please revert it from your diff. If you rebase on master, this should be "b", I just fixed this bug (see the blame for that file).